docker-datadog-dual-shipping-example/env-example

```diff
@@ -0,0 +1,16 @@
+# Required
+DD_API_KEY=<YOUR DD KEY>
+DD_APM_ADDITIONAL_ENDPOINTS={"https://in-datadog.codesee.io": ["<CodeSee API Key here>"]}
```

Comment:
Is this an example `.env` file? I didn't know you could put `{` and `"` and whitespace in a value like this!

Reply:
yep it parsed correctly on my actual .env

Reply:
Just to be clear, if someone already has datadog agents installed in their infra, the only change they need to make is to add this one env var? And it will send data to both Datadog and CodeSee?

Reply:
correct and just need agent to be: Requires Agent version >= 6.17 or 7.17.